pkg/gce-pd-csi-driver/utils.go

```diff
@@ -56,14 +57,30 @@ func NewNodeServiceCapability(cap csi.NodeServiceCapability_RPC_Type) *csi.NodeS
 	}
 }
 
+// Reflect magic below simply clears Secrets map from request.
+func clearSecrets(req interface{}) interface{} {
```

Comment:
@jsafrane how do you feel about putting this into csi-lib-utils as an alternative to protosanitizer so that it is not specific to a csi driver?

Reply:
Sounds good to me, but with a huge warning that only field explicitly named `Secrets` is cleared.